Jenkinsfile

```diff
@@ -151,7 +151,7 @@ String unit_packages() {
                            'libpmemobj protobuf-c ' +
                            'spdk-devel libfabric-devel ' +
                            'pmix numactl-devel ' +
-                           'libipmctl-devel ' +
+                           'libipmctl-devel python36-pyxattr ' +
```

Comment:
if we are adding a dependency we should update the docker files:
https://github.com/daos-stack/daos/tree/master/utils/docker

Reply:
I added it to the CentOS one already as that's used for pylint and in CI but I haven't added it to the others.  I doubt that ubuntu and centos 8 have enough packages to run testing in master anyway but I'll check.

Reply:
ah i missed the centos7 one. but anyway it's not about CI testing, but many users refer to those for installing dependencies, so we should keep them all up to date.

Reply:
Yes, this will only affect the ability to run NLT.  I'm running a test now but it'll take a while to build all the images.

Reply:
Done.  I had to add PyYAML on a couple of them as well.